tox.ini

```diff
@@ -15,10 +15,10 @@ commands =
     {posargs}
 passenv = GOOGLE_APPLICATION_CREDENTIALS
 
-[testenv:py38-integration]
+[testenv:py310-integration]
```

Comment:
I can't figure out how to leave a comment on line 2, but should the envlist be updated similarly?

Reply:
yes, good catch